crates/sui/src/sui_commands.rs

```diff
@@ -339,8 +343,22 @@ impl SuiCommand {
             SuiCommand::Move {
                 package_path,
                 build_config,
+                client_config,
+                accept_defaults,
                 cmd,
-            } => execute_move_command(package_path.as_deref(), build_config, cmd),
+            } => {
+                let config_path =
+                    client_config.unwrap_or(sui_config_dir()?.join(SUI_CLIENT_CONFIG));
+                prompt_if_no_config(&config_path, accept_defaults).await?;
+                let context = WalletContext::new(&config_path, None, None)?;
+                let chain_id = context
```

Comment:
As I understand it, this is querying the network but what happens if there is no network connectivity? Does it failover gracefully and uses `None` for `chain_id` or we are getting problems with off-line execution of Move commands?